docs/ReleaseNotes/ReleaseNotes.Atom.May.2019.md

```diff
@@ -0,0 +1,32 @@
+# Atom Release (May 2019)
+
+!!! warning
+	- Due to significant data format changes, you must perform a full install of the Atom version. You cannot perform an upgrade process and retain your existing data. 
+	- To copy over your data to a CM-Well node with the new version, either re-ingest your data, or perform a DC-Sync process between the "old node" and the "new node". 
```

Comment:
Please replace "node" with "cluster". Moreover, we need to emphasize the need of a temp. cluster in the case of DC-Sync. It's good to state the the preferable option is re-ingest, but in cases when users cannot afford that, they need a temp. cluster like so: let's say A is the existing Zerba Cluster. and T is a temp. cluster with either Zebra or Atom but no data. Steps are: 1. DC-Sync A->T. 2. "wipe-and-install" Atom on A. 3. DC-Sync T->A.

Reply:
Added.